jekyll/_cci2/faq.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: classic-docs
+title: "Frequently Asked Questions"
+short-title: "FAQ"
+categories: [getting-started]
+---
+
+## Does CircleCI 2.0 run inference commands?
+
+No, CircleCI 2.0 does not infer anything from your project.
+
+This means that no commands are run automatically. Don’t assume that a particular library/package is available unless it is bundled with the Docker image you are using!
+
+## My project is running on CircleCI 2.0, but the build is frozen!
+
+Builds often freeze due to syntax errors in `circle.yml`.
+
+Cancel the build, check your `circle.yml` for proper indents, and ensure that all stages/steps have the required keys.
+
+## Can I use CircleCI 2.0 without creating base images?
+
+Yes, you can use* one of ours!
+
+The `circleci/build-image:ubuntu-14.04-XL-922-9410082` image has the same content as the Ubuntu Trusty 14.04 image our web app uses. Just know that the image is fairly large (around 17.5 GB uncompressed), so it’s less ideal for local testing.
+
+The image defaults to running actions as the `ubuntu` user and is designed to work with network services provided by Docker Compose.
+
+Here’s a [list of languages and tools](https://circleci.com/docs/build-images-2-0/) included in the image.
```

Comment:
This should be changed to `[list of languages and tools]({{site.baseurl}}/docs/build-images-2-0/)`.